docs/_articles/en/getting-started/java-setup.md

```diff
@@ -27,6 +27,8 @@ Inside Visual Studio Code, you will need to set the `salesforcedx-vscode-apex.ja
 
 MacOS:
 
+For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

Comment:
```suggestion
For Mac OS X 10.5 or later, run `/usr/libexec/java_home` in your Terminal to get the default JDK location. To find all installed JDKs, use `/usr/libexec/java_home -V`.
```

Reply:
I did a quick peek, and I know we aren't totally consistent. So I found some Apple dev docs online, and they capitalize "Terminal" so we'll just fix them as we run across it.

Reply:
Sounds good, I'll make the change